internal/kuberesource/parts.go

```diff
@@ -553,32 +553,3 @@ func ServiceMeshProxy() *applycorev1.ContainerApplyConfiguration {
 			"-l", "debug",
 		)
 }
-
-// CryptsetupInitCommand returns the init command for the cryptsetup
```

Comment:
Now that we provide a container image and command, you can replace the documentation at https://github.com/edgelesssys/contrast/blob/6791f9e5c244889e8dedb67edb8cae33a6318985/docs/docs/architecture/secrets.md?plain=1#L101-L104 and instead explain how to do this with the initializer image.